features/support/env.rb

```diff
@@ -44,10 +44,15 @@
 end
 
 Capybara.register_driver :selenium do |app|
-  profile = Selenium::WebDriver::Firefox::Profile.new
+  browser = ENV['TARGET_BROWSER'].to_sym || :firefox
```

Comment:
Could we maybe put this in a private method along with `browser == :chrome` -> Probably call the second method `chrome?`

Reply:
@luke-hill , Thank you to review.

I changed it, is it such an image? > https://github.com/natritmeyer/site_prism/pull/214/commits/fe9afdda11841a75873eec8408ced035756dab6b